### ISSUE TITLE
Elevation unlikely

### ISSUE THREAD

**Issue by MortenHofft** [gbif/portal-feedback#3407]
https://www.gbif.org/occurrence/search?issue=ELEVATION_UNLIKELY&advanced=1

This flag is currently not applied to any records. But we have many records with an elevantion above 10km. Didn't we use to have a threshold that everything above 9999 meters was considered unlikely (I believe elevation refers to a point on the surface, and so birds flying at >11km altitude would be captured differently)

**Comment by ManonGros** [comment]
Yes I thought we did too. There are still a lot of records unflagged though (and not just birds - see [these plant records](https://www.gbif.org/occurrence/search?elevation=9370,9999&taxon_key=6&advanced=1)).
